tests/validation/tests/v3_api/common.py

```diff
@@ -891,10 +891,11 @@ def validate_http_response(cmd, target_name_list, client_pod=None,
                 wget_cmd = "wget -qO- " + cmd
             result = kubectl_pod_exec(client_pod, wget_cmd)
             result = result.decode()
-        result = result.rstrip()
-        assert result in target_name_list
-        if result in target_hit_list:
-            target_hit_list.remove(result)
+        if result is not None:
```

Comment:
What when the result is None? Can we print something, because this happens a lot of time (not sure why). Do we know why the result is None sometimes?

Reply:
From my observation, it happens when the curl command is sent right after the URL is accessible but the pod is not ready to respond the request.